src/tests/ftest/util/general_utils.py

```diff
@@ -580,3 +581,20 @@ def check_uuid_format(uuid):
     """
     pattern = re.compile("([0-9a-fA-F-]+)")
     return bool(len(uuid) == 36 and pattern.match(uuid))
+
+
+def get_remote_file_size(host, file_name):
+    """Obtain remote file size.
+
+      Args:
+        file_name (str): name of remote file
+
+      Returns:
+        integer value of file size
+    """
+
+    cmd = "ssh" " {}@{}" " stat -c%s {}".format(
+        getuser(), host, file_name)
+    result = run_command(cmd)
+
+    return int(result.stdout)
```

Comment:
Since it is a generic method, I'd recommend to be cautious before returning int(result.stdout). It's a good to validate the result before converting to int. What happens if the file_name doesn't exist in the remote host ? Doing int(result.stdout) may throw an exception in that case. If possible, check for exit code and proceed only if it is 0.

Reply:
I would say, if there is any issue while running the run_comand(cmd), run_command() will automatically raise a DaosTestError exception.